.github/scripts/import_strings.py

```diff
@@ -51,20 +55,18 @@ def getReferenceFilesToml(toml_path, reference_locale):
     """Extract list of reference files from project configuration (TOML)"""
 
     basedir = os.path.dirname(toml_path)
-    project_config = paths.TOMLParser().parse(toml_path, env={"l10n_base": ""})
-    basedir = os.path.join(basedir, project_config.root)
+    project_config_paths = L10nConfigPaths(
+        toml_path, locale_map={"android_locale": get_android_locale}
+    )
```

Comment:
Strictly speaking the `locale_map` doesn't need to be set here as the target paths are never referenced, but it doesn't hurt either.